engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -265,6 +290,30 @@ public ModuleFactory getModuleFactory() {
     }
 
     public PermissionProvider getPermissionProvider(Module module) {
-        return permissionProviderFactory.createPermissionProviderFor(module, module.getClassPredicate());
+        // TODO: This has only one usage. Audit to see if it's worth keeping.
+        return permissionProviderFactory.createPermissionProviderFor(
+                module,
+                x -> false  // modules (other than engine) never contain classpath classes
+        );
+    }
+
+    /** Convert URL to File, with support for {@code jar} URLs. */
+    private static File urlToFile(URL url) {
```

Comment:
Do we have any existing utility modules where this could go?

(I took a quick peek in o.t.e.utilities, but got spooked by the cobwebs.)

Reply:
@skaldarnar and I also had a short conversation about o.t.e.utils...
I personally am okay with some "utils" or "common" rummage box to throw things into that don't really fit anywhere else and would live a lonely life if put in some area of the code dedicated to them. Naturally such a rummage box needs cleaning up every once in a while in case some of the things in there suddenly become clusterable due to new similar things being dropped in there.
Skal seemed more inclined to have them live a lonely life in an area dedicated to them and similar things that we can envision being added in the mid-term future.